### DISCUSSION TITLE
find_unused_parameters in the lightning trainer (1.3.2)

### DISCUSSION THREAD

**Discussion by shamanez** [Lightning-AI/pytorch-lightning#7796] [DDP / multi-GPU / multi-node] [answered]
## 🐛 Bug

If there are unsued parameters in the model, should we still explicitly mentioned the trainer as follows?

**plugins=[DDPPlugin(find_unused_parameters=True)]**

` trainer = pl.Trainer.from_argparse_args(
args, weights_summary=None, 
callbacks=[logging_callback],  logger=logger,        
plugins=[DDPPlugin(find_unused_parameters=True)],
**train_params,)`

**Comment by awaelchli** [comment, marked as answer]
No, it is set to True by default and you want to turn it off unless you need it :) 

Reference:
https://pytorch-lightning.readthedocs.io/en/1.3.3/benchmarking/performance.html#when-using-ddp-set-find-unused-parameters-false

**Comment by shamanez** [comment]
Thanks. Git it
